get_hrc.py

```diff
@@ -108,6 +100,18 @@ def format_proton_data(dat, descrs):
     newdat['hhmm'] = np.array([f"{t.hour}{t.minute:02}" for t in times.datetime]).astype(int)
     newdat['p11'] = np.full(len(times), -1.0e5)
 
+    # Add the other channel data marking as 1.0e5 if missing
+    for t in tabs:
+        # Take the second return of intersect1d and make a mask of good data
+        ok = np.zeros(len(time_ref)).astype(bool)
+        idx_ok = np.intersect1d(time_ref, t['time_tag'], assume_unique=True,
+                                return_indices=True)[1]
```

Comment:
I note that this use of return_indices requires a numpy greater than the one in flight ska3.